app/src/main/java/a3/com/convo/activities/LoginActivity.java

```diff
@@ -34,28 +34,30 @@
 import java.util.HashMap;
 import java.util.List;
 
+import a3.com.convo.Constants;
 import a3.com.convo.Models.Page;
 import a3.com.convo.R;
 
 public class LoginActivity extends AppCompatActivity {
 
-    LoginButton loginButton;
-    CallbackManager callbackManager;
-    Activity context;
+    private LoginButton loginButton;
+    private CallbackManager callbackManager;
+    private Context context;
```

Comment:
You don't need to save this as a member variable as long as you have access to activity or fragment